src/org/labkey/test/tests/ProjectSettingsTest.java

```diff
@@ -86,53 +160,167 @@ public static void setupProject()
 
     protected void setUpTest()
     {
-        _containerHelper.createProject(getProjectName(), null);
-        checkHelpLinks(null, true, true);
-        _containerHelper.createProject(getProjectAlteredName(), null);
-        checkHelpLinks(null, true, true);
-
-        goToProjectSettings(getProjectAlteredName());
-        setFormElement(Locator.name("reportAProblemPath"), "");
-        clickButton("Save");
-        assertElementNotPresent(Locators.labkeyError);
-
-        checkHelpLinks(getProjectAlteredName(), false, true);
-//        assertElementNotPresent("Support link still present after removing link from settings", supportLink);
+        _containerHelper.deleteProject(PROJ_CHANGE, false);
+        _containerHelper.deleteProject(PROJ_BASE, false);
+
+        // Make sure the site settings are reset.
+        resetSiteSettings();
+        _containerHelper.createProject(PROJ_CHANGE, null);
+        _containerHelper.createProject(PROJ_BASE, null);
+    }
+
+    private static final String DT_LIST_NAME = "Date_And_Time";
+    private static final String DT_LIST_ID_COL = "id";
+    private static final String DT_LIST_DATE_COL = "Date";
+    private static final String DT_LIST_TIME_COL = "Time";
+    private static final String DT_LIST_DATETIME_COL = "DateTime";
+
+    private void createDateAndTimeList(String project, List<Map<String, String>> listData)
+    {
+        _listHelper.createList(project, DT_LIST_NAME, ListHelper.ListColumnType.AutoInteger, DT_LIST_ID_COL,
+                new FieldDefinition(DT_LIST_DATE_COL, FieldDefinition.ColumnType.Date),
+                new FieldDefinition(DT_LIST_TIME_COL, FieldDefinition.ColumnType.Time),
+                new FieldDefinition(DT_LIST_DATETIME_COL, FieldDefinition.ColumnType.DateAndTime));
+
+
+        // Use the default format to initially populate the lists.
+        StringBuilder bulkImportData = new StringBuilder();
+        bulkImportData.append(String.format("%s\t%s\t%s\n", DT_LIST_DATE_COL, DT_LIST_TIME_COL, DT_LIST_DATETIME_COL));
+
+        for(Map<String, String> listRow : listData)
+        {
+            bulkImportData.append(String.format("%s\t%s\t%s\n",
+                    listRow.get(DT_LIST_DATE_COL),
+                    listRow.get(DT_LIST_TIME_COL),
+                    listRow.get(DT_LIST_DATETIME_COL))
+            );
+        }
+
+        _listHelper.bulkImportData(bulkImportData.toString());
+
     }
```

Comment:
This isn't a list or domain designer focused test, it shouldn't spend a bunch of time in UI that is tested elsewhere. Use API helpers for list creation (`IntListDefinition`) and data import (`TestDataGenerator`).

Reply:
Changed.